packages/ketchup/src/managers/kup-data/kup-data-declarations.ts

```diff
@@ -87,7 +87,7 @@ export interface KupCommand {
  * Generic cell interface.
  */
 export interface KupDataCell {
-    value: string;
+    value?: string;
```

Comment:
Why?
value should be mandatory

Reply:
It creates problems in webupjs because it is optional there